packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -1964,7 +1977,7 @@ test('adding metrics to reordered table keeps order', function(assert) {
 
   andThen(() => {
     assert.deepEqual(
-      find('.table-header-cell__title')
+      find('div.table-header-row-vc .table-header-cell__title')
```

Comment:
Do we need to be this specific for selectors in this file?

Reply:
Same answer as above